pkg/controllers/managementuser/resourcequota/resource_quota_common.go

```diff
@@ -236,17 +250,6 @@ func getProjectID(ns *corev1.Namespace) string {
 	return ""
 }
 
-func getProjectNamespaceName(projectID string) (string, string) {
-	if projectID == "" {
-		return "", ""
-	}
-	parts := strings.Split(projectID, ":")
-	if len(parts) == 2 {
-		return parts[0], parts[1]
-	}
-	return "", ""
-}
```

Comment:
`ref.Parse` is slightly different than this: https://github.com/rancher/rancher/blob/c068d30eaffa24337b97e16e3a12d22219d0ac6d/pkg/ref/parse.go#L33-L37

However, I don't think the specific edge case that the helper function is catching merits another function that needs to be defined.

On another note, we probably need to clean up the code at some point to consolidate all these one-off helper functions that are doing redundant stuff (e.g. we put labels and annotation stuff for projects all across several controllers rather than referencing a single package that contains the expected label + annotation).

Reply:
I'd file an issue for that.

Reply:
https://github.com/rancherlabs/engineering-improvements/issues/30